kernel/declarations.ml

```diff
@@ -176,6 +177,8 @@ type one_inductive_body = {
 
     mind_recargs : wf_paths; (** Signature of recursive arguments in the constructors *)
 
+    mind_relevant : Sorts.relevance;
```

Comment:
Why not `mind_relevance` to be uniform with`const_relevance`?

Reply:
Changed